automatic/ssdt17/ssdt17.nuspec

```diff
@@ -12,7 +12,21 @@
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
         <tags>SQL Server Data Tools SSDT Visual Studio 2017 admin</tags>
 		<summary>This update addresses product defects for SQL Server projects, database deployments, and database tools.</summary>
-		<description>This update addresses product defects for SQL Server projects, database deployments, and database tools.</description>
+		<description>
+#### Package Parameters
+The following package parameters can be set:
+
+ * `/analysis[:VSInstanceIds]` - Install SSAS to specified VS2017 instances.
+ * `/integration[:VSInstanceIds]` - Install SSIS to specified VS2017 instances.
+ * `/reporting[:VSInstanceIds]` - Install SSRS to specified VS2017 instances.
+
+VSInstanceIds is the list of target VS 2017 instance ids seperated by comma. It can be absent when there is only one VS 2017 
+instance installed on current machine
+
+To pass parameters, use --params "''" (e.g. choco install ssdt17 [other options] --params="'/analysis /reporting:'").
```

Comment:
Does the example have an extra colon after `/reporting`?

Reply:
Typo. As an example it should have some VS2017 instance names, like /reporting:VsInstance1,VsInstance2

Thanks for the review!